app.rb

```diff
@@ -121,13 +130,23 @@ def create_rental()
 
     colorize_outprint(36, 'Date: ')
     date = gets.chomp
+
     Rental.new(date, @books[book_option], @people[person_option])
 
+    new_rental = {
+      dating: date,
+      book_num: book_option,
+      person_num: person_option
+    }
+    @rentals.push(new_rental)
+
     colorize_output(36, 'Rental created successfully')
+    puts
   end
 
   def list_rentals_of_person_id()
-    colorize_outprint(36, 'Id of person: ')
+    list_all_people
+    colorize_outprint(36, 'Enter Id of person from above: ')
```

Comment:
- [ ] Great job using colorized messages, I really liked it. 👍

Reply:
Hi @Omar-Muhamad, thanks for the review and comments.